core/src/types.ts

```diff
@@ -266,3 +266,40 @@ export declare interface DebtAuctionEnriched extends DebtAuctionActive {
 }
 
 export declare interface DebtAuctionTransaction extends DebtAuctionEnriched, CompensationAuctionTransactionFees {}
+
+type CollateralType = CollateralConfig['title'];
+export declare interface Collateral {
+    type: CollateralType;
```

Comment:
Please always use `collateralType` key for collateral type for consistency. I also don't understand why `Vault` need to have two keys supposedly containing the same thing? `type` and `collateralType`

Reply:
extracted collateral type into the separate interface.